interactors/src/main/kotlin/interactors/StudentsInteractor.kt

```diff
@@ -0,0 +1,22 @@
+package com.students.results.interactors
+
+import arrow.core.Either
+import arrow.core.flatMap
+import com.students.results.repository.ExamsRepository
+import com.students.results.repository.StudentsRepository
+import com.students.results.services.NotateExamException
+import com.students.results.services.Students
+import com.students.results.services.requests.NotateExam
+
+class StudentsInteractor(private val studentsRepository: StudentsRepository, private val examsRepository: ExamsRepository) : Students {
+
+    override fun notate(notateExam: NotateExam): Either<NotateExamException, Unit> =
+            studentsRepository.findStudentById(studentId = 1).flatMap { student ->
```

Comment:
your studentId is defined in the notateExam class, we should probably use that otherwise your student 1 will get all the grades

Reply:
I already noticed it in my comment, but thx